stable/pipelines/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: pipelines
 home: https://jfrog.com/pipelines/
-version: 1.4.3
+version: 2.0.0
```

Comment:
Please change version to patch 1.4.4

Reply:
@rimusz  This is a breaking change for imagePull secrets  and upgrade to 12.x postgresql - Hence added Major version bump

Reply:
ok